### ISSUE TITLE
`plotPopulationTimeProfile` range color does not match line color if multiple results are assigned to the same group

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/OSPSuite-R#1064]
Both shaded areas should have the color of the line (blue). They will not be distinguishable, but that's fine.

```
simFilePath <- system.file("extdata", "Aciclovir.pkml", package = "ospsuite")
sim <- loadSimulation(simFilePath)

outputPath <- c("Organism|PeripheralVenousBlood|Aciclovir|Plasma (Peripheral Venous Blood)",
                "Organism|Muscle|Intracellular|Aciclovir|Concentration")

simResults <- importResultsFromCSV(simulation = sim, filePaths = system.file("extdata", "SimResults_pop.csv", package = "ospsuite"))

obsData <- lapply(c("ObsDataAciclovir_1.pkml", "ObsDataAciclovir_3.pkml"),
                  \(x){loadDataSetFromPKML(system.file("extdata", x, package = "ospsuite"))})

names(obsData) <- lapply(obsData, \(x){x$name})


myDataCombined <- DataCombined$new()
# Add simulated results
myDataCombined$addSimulationResults(simulationResults = simResults,
                                    quantitiesOrPaths = outputPaths,
                                    groups = "Aciclovir PVB")

plotPopulationTimeProfile(myDataCombined)
```

![image](https://user-images.githubusercontent.com/25903040/183635103-68cfb263-09b8-4acc-b787-fb7f3d3870a6.png)

**Comment by IndrajeetPatil** [comment]
Tried a few things, but I can't seem to solve this. 

If I map so that the ribbons have the same color, somehow the lines get joined to make a saw blade like shape. Need to talk to Pierre about this.

``` r
library(ospsuite)
#> Loading required package: rClr
#> Loading the dynamic library for Microsoft .NET runtime...
#> Loaded Common Language Runtime version 4.0.30319.42000

simFilePath <- system.file("extdata", "Aciclovir.pkml", package = "ospsuite")
sim <- loadSimulation(simFilePath)

outputPaths <- c(
  "Organism|PeripheralVenousBlood|Aciclovir|Plasma (Peripheral Venous Blood)",
  "Organism|Muscle|Intracellular|Aciclovir|Concentration"
)

simResults <- importResultsFromCSV(
  simulation = sim,
  filePaths = system.file("extdata", "SimResults_pop.csv", package = "ospsuite")
)

myDataCombined <- DataCombined$new()

myDataCombined$addSimulationResults(
  simulationResults = simResults,
  quantitiesOrPaths = outputPaths,
  groups = "Aciclovir PVB"
)

plotPopulationTimeProfile(myDataCombined)
```

![](https://i.imgur.com/meJXjqR.png)

<sup>Created on 2022-08-09 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1.9000)</sup>

**Comment by IndrajeetPatil** [comment]
Needs to be resolved in `{tlf}` first: 
https://github.com/Open-Systems-Pharmacology/TLF-Library/issues/360

**Comment by IndrajeetPatil** [comment]
This should be fixed in #1054 

``` r
library(ospsuite)
#> Loading required package: rClr
#> Loading the dynamic library for Microsoft .NET runtime...
#> Loaded Common Language Runtime version 4.0.30319.42000

simFilePath <- system.file("extdata", "Aciclovir.pkml", package = "ospsuite")
sim <- loadSimulation(simFilePath)

outputPaths <- c(
  "Organism|PeripheralVenousBlood|Aciclovir|Plasma (Peripheral Venous Blood)",
  "Organism|Muscle|Intracellular|Aciclovir|Concentration"
)

simResults <- importResultsFromCSV(
  simulation = sim,
  filePaths = system.file("extdata", "SimResults_pop.csv", package = "ospsuite")
)

myDataCombined <- DataCombined$new()

myDataCombined$addSimulationResults(
  simulationResults = simResults,
  quantitiesOrPaths = outputPaths,
  groups = "Aciclovir PVB"
)

plotPopulationTimeProfile(myDataCombined)
```

![](https://i.imgur.com/aRJf7HA.png)

<sup>Created on 2022-08-10 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1.9000)</sup>
